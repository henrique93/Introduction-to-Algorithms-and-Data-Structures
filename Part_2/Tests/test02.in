10
send AAAA #AAAA1 AAAAAAAAAAAAAA1
send AAAA #AAAA2 AAAAAAAAAAAAAA2
send AAAA #AAAA3 AAAAAAAAAAAAAA3
send AAAA #AAAA4 AAAAAAAAAAAAAA4
send AAAA #AAAA5 AAAAAAAAAAAAAA5
send AAAA #AAAA6 AAAAAAAAAAAAAA6
send AAAA #AAAA7 AAAAAAAAAAAAAA7
send AAAA #AAAA8 AAAAAAAAAAAAAA8
send AAAA #AAAA9 AAAAAAAAAAAAAA9
send AAAA #AAA10 AAAAAAAAAAAAA10
list #AAAA3
list #AAAA9
list #AAA10
list #AAAA1
list #AAAA5
list #AAAA8
list #AAAA4
list #AAAA2
list #AAAA7
list #AAAA6
quit
