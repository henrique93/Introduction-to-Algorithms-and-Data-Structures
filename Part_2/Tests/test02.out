AAAA #AAAA3 : AAAAAAAAAAAAAA3
AAAA #AAAA9 : AAAAAAAAAAAAAA9
AAAA #AAA10 : AAAAAAAAAAAAA10
AAAA #AAAA1 : AAAAAAAAAAAAAA1
AAAA #AAAA5 : AAAAAAAAAAAAAA5
AAAA #AAAA8 : AAAAAAAAAAAAAA8
AAAA #AAAA4 : AAAAAAAAAAAAAA4
AAAA #AAAA2 : AAAAAAAAAAAAAA2
AAAA #AAAA7 : AAAAAAAAAAAAAA7
AAAA #AAAA6 : AAAAAAAAAAAAAA6
